docs/application/dotnet/get-started/iot/first-app.md

```diff
@@ -0,0 +1,219 @@
+        </StackLayout>
+    </ContentPage.Content>
+</ContentPage>
+```
+
+This application is constructed with the following Xamarin.Forms controls:
+
+- There is the `ContentPage` where you set up the view to display. You assign what you want to display to the `Content` property of the ContentPage.
+- The `StackLayout` is added. This layout positions ("stacks") its child elements in a single vertical (default) or horizontal line.
+- The `Label` is added, whose `Text` property is the message you want to display.
+
+The code does not show the instantiation of the `App` class. Since application launching is platform-specific, the launching part, including the instantiation of the `App` class and the declaration of the `Main()` function (required as the entry point of every C\# program), happens in the \<projectname\>.Tizen project, in the matching file there. The following example shows the content of the `<projectname>.Tizen.cs` file:
+
+```csharp
+using System;
+using Xamarin.Forms;
+
+namespace CrossTemplate1
+{
+    class Program : global::Xamarin.Forms.Platform.Tizen.FormsApplication
+    {
+        protected override void OnCreate()
+        {
+            base.OnCreate();
+
+            LoadApplication(new App());
+        }
+
+        static void Main(string[] args)
+        {
+            var app = new Program();
+            Forms.Init(app);
+            app.Run(args);
+        }
+    }
+}
+```
+
+## Package Your Application
+
+A Tizen .NET application is deployed in the form of an installable package, with the package file extension `.tpk`. A Tizen .NET package has a relatively simple structure: internally it is a ZIP file with content that matches the directory layout of the project.
+
+The package contains the following:
+
+- The `shared` directory, which is for items that are considered system-wide (shareable).
+
+    The application icon is packaged in the `shared/res` directory on installation, and the icon appears on the home screen with the icons for the other applications. You can either replace the icon (which is just a copy of the default Tizen logo) with one of your own using the file name generated by Visual Studio, or put a new icon in the same project directory and update the package manifest to indicate the new name.
+
+- The `res` directory, which is for application-private resources.
+
+    If the application needs a file to open at runtime, it can be placed here.
+
+- The `bin` directory, which contains the generated application executable.
+- The `lib` directory, which contains the generated application support code.
+
+    If you use nugget libraries, they are imported in the lib directory.
+
+- The package manifest, which defines the application properties and is used at the installation time.
+- Two signature files (author and distributor), which are checked at the installation time.
+
+The following figure shows the layout of the platform-specific (Tizen) project. 
```

Comment:
In other get started guides, it is a period so for maintaining consistency we can keep it as is. 

https://docs.tizen.org/application/dotnet/get-started/wearable/first-app/

Kindly replace the period . with : colon at the end of the sentence. This is as per Tizen Style guide as word **following** is in the sentence. 
```suggestion 
The following figure shows the layout of the platform-specific (Tizen) project:
```

Reply:
Fixed.